api/orchestration_movers.go

```diff
@@ -528,3 +528,44 @@ func (o *datasyncOrchestrator) datamoverRunDescribe(ctx context.Context, group,
 		Result:                   exec.Result,
 	}, nil
 }
+
+// startTaskRun starts the execution for a given task
+func (o *datasyncOrchestrator) startTaskRun(ctx context.Context, group, name string) (*datasync.StartTaskExecutionOutput, error) {
```

Comment:
The orchestrator functions should only return the thing we care about, in this case just the `string` id of the new run, not the entire `StartTaskExecutionOutput`

Reply:
done.